packages/components/src/Components/ConditionalFilter/conditional-filter.scss

```diff
@@ -5,12 +5,20 @@
         @include rem('max-width', 150px);
     }
 
-    .ins-c-conditional-filter__group .pf-c-dropdown__toggle-text {
-        width: 102px;
-        text-align: left;
+    .ins-c-conditional-filter__group {
+        .pf-c-dropdown__toggle-text {
+            width: 102px;
+            text-align: left;
 
-        & > svg {
-            margin-right: var(--pf-global--spacer--xs);
+            & > svg {
+                margin-right: var(--pf-global--spacer--xs);
+            }
+        }
+
+        &.no__label {
+            .pf-c-dropdown__toggle-text {
```

Comment:
Since this is effectively a single selector can you make align it on one line? `&.no__label .pf-c-dropdown__toggle-text {`